core/src/vdaf.rs

```diff
@@ -176,16 +176,18 @@ impl VdafInstance {
     }
 }
 
-impl TryFrom<&taskprov::VdafType> for VdafInstance {
+impl TryFrom<&taskprov::VdafConfig> for VdafInstance {
     type Error = &'static str;
 
-    fn try_from(value: &taskprov::VdafType) -> Result<Self, Self::Error> {
+    fn try_from(value: &taskprov::VdafConfig) -> Result<Self, Self::Error> {
         match value {
-            taskprov::VdafType::Prio3Count => Ok(Self::Prio3Count),
-            taskprov::VdafType::Prio3Sum { bits } => Ok(Self::Prio3Sum {
-                bits: *bits as usize,
+            taskprov::VdafConfig::Prio3Count => Ok(Self::Prio3Count),
+            taskprov::VdafConfig::Prio3Sum {
+                max_measurement: _max_measurement,
+            } => Ok(Self::Prio3Sum {
+                bits: 32, // TODO(#3436): plumb through max_measurement once it's available
```

Comment:
This will certainly need to change, but at the time I'm writing this, the change to `libprio-rs` to specify `max_measurement` instead of `bits` for Prio3Sum is still pending. I'll update this once the `libprio-rs` lands.